libraries/exoplayer_rtsp/src/main/java/androidx/media3/exoplayer/rtsp/reader/RtpH265Reader.java

```diff
@@ -0,0 +1,317 @@
+    //    |                                                               |
+    //    |                  NAL unit payload data                        |
+    //    |                                                               |
+    //    |                               +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    //    |                               :...OPTIONAL RTP padding        |
+    //    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+
+    int numBytesInData = data.bytesLeft();
+    fragmentedSampleSizeBytes += writeStartCode();
+    trackOutput.sampleData(data, numBytesInData);
+    fragmentedSampleSizeBytes += numBytesInData;
+
+    int nalHeaderType = (data.getData()[0] >> 1) & 0x3F;
+    bufferFlags = getBufferFlagsFromNalType(nalHeaderType);
+  }
+
+
+  /**
+   * Processes STAP Type A packet (RFC7798 Section 4.4.2).
+   *
+   * <p>Outputs the received aggregation units (with start code prepended) to {@link #trackOutput}.
+   * Sets {@link #bufferFlags} and {@link #fragmentedSampleSizeBytes} accordingly.
+   */
+  @RequiresNonNull("trackOutput")
+  private void processSingleTimeAggregationPacket(ParsableByteArray data) throws ParserException {
+    //  An Example of an AP Packet Containing Two Aggregation
+    //  Units without the DONL and DOND Fields.
+    //      0                   1                   2                   3
+    //     0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+    //    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    //    |                          RTP Header                           |
+    //    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    //    |    PayloadHdr (Type=48)       |         NALU 1 Size           |
+    //    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    //    |     NALU 1 HDR                |                               |
+    //    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-       NALU 1 Data              |
+    //    |                                                               |
+    //    |                                                               |
+    //    +               +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    //    |               | NALU 2 Size                   | NALU 2 HDR    |
+    //    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    //    | NALU 2 HDR    |                                               |
+    //    +-+-+-+-+-+-+-+-                    NALU 2 Data                 |
+    //    |                                                               |
+    //    |                               +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    //    |                               :...OPTIONAL RTP padding        |
+    //    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+
+    throw ParserException.createForMalformedManifest(
+        "need to implement processSingleTimeAggregationPacket",
```

Comment:
Do you have an ETA for this?

Reply:
I'm not sure "not finding a server to test" is a valid reason that you "can't implement it". Please provide an implementation following the RFC, and if possible, do some basic tests like feeding some human generated packets to it.

Reply:
Added a TODO for it, we will visit again to complete it.